saxml/tools/docker/Dockerfile.admin

```diff
@@ -42,18 +42,21 @@ ENV SAX_CELL=/sax/test
 # Use "10000" as default port for admin server gRPC endpoint.
 ENV PORT=10000
 
+ENV SAX_ROOT=gs://${GSBUCKET}/sax-root
+ENV FS_ROOT=gs://${GSBUCKET}/sax-fs-root
+
 # Create a script that configures and runs the admin server.
 RUN echo '#!/bin/bash\n\
 admin_config \
   --sax_cell=${SAX_CELL} \
-  --sax_root=gs://${GSBUCKET}/sax-root \
-  --fs_root=gs://${GSBUCKET}/sax-fs-root \
+  --sax_root=${SAX_ROOT} \
+  --fs_root=${FS_ROOT} \
   --alsologtostderr\n\
 admin_server \
   --sax_cell=${SAX_CELL} \
-  --sax_root=gs://${GSBUCKET}/sax-root \
+  --sax_root=${SAX_ROOT} \
   --port=${PORT} \
   --alsologtostderr\n' \
 > /usr/bin/admin_server_entrypoint.sh && chmod +x /usr/bin/admin_server_entrypoint.sh
 
-ENTRYPOINT ["/usr/bin/admin_server_entrypoint.sh"]
+ENTRYPOINT ["/usr/bin/admin_server_entrypoint.sh"]
```

Comment:
Wondering what the change is on this line here.

Reply:
Should be fixed on the new commit